_rules/table-header-cells-refer-to-data-cells-d0f69e.md

```diff
@@ -0,0 +1,236 @@
+---
+id: d0f69e
+name: All table header cells have corresponding data cells
+rule_type: atomic
+description: |
+  This rule checks that each table header has corresponding data cells in a table element.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+- DOM Tree
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Applicability
+
+This rule applies to `th` elements within a `table`, where the `table` is [visible](#visible) and [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
"within a `table`" is a bit ambiguous. It is something we have worked a lot with in other rules (therefore some of them say stuff like "descendant in the flat tree").

Reply:
Suggestion for applicability: 

```
This rule applies to each `th` element that
- is [visible](#visible) and [included in the accessibility tree](#included-in-the-accessibility-tree)
- is a [descendant]() of a `table` element that is [visible](#visible) and [included in the accessibility tree](#included-in-the-accessibility-tree), and
- is either a [column header](https://html.spec.whatwg.org/#column-header) or [row header](https://html.spec.whatwg.org/#row-header).
```